crates/router/src/connector/payme.rs

```diff
@@ -843,11 +841,34 @@ impl api::IncomingWebhook for Payme {
                     .change_context(errors::ConnectorError::WebhookBodyDecodingFailed)
             }
             transformers::NotifyType::SaleChargeback
-            | transformers::NotifyType::SaleChargebackRefund => {
-                Err(errors::ConnectorError::WebhookEventTypeNotFound).into_report()
-            }
+            | transformers::NotifyType::SaleChargebackRefund => serde_json::to_value(resource)
+                .into_report()
+                .change_context(errors::ConnectorError::WebhookBodyDecodingFailed),
         }?;
 
         Ok(res_json)
     }
+
+    fn get_dispute_details(
+        &self,
+        request: &api::IncomingWebhookRequestDetails<'_>,
+    ) -> CustomResult<api::disputes::DisputePayload, errors::ConnectorError> {
+        let webhook_object =
+            serde_urlencoded::from_bytes::<payme::WebhookEventDataResource>(request.body)
+                .into_report()
+                .change_context(errors::ConnectorError::WebhookBodyDecodingFailed)?;
+
+        Ok(api::disputes::DisputePayload {
+            amount: webhook_object.price.to_string(), //check
+            currency: webhook_object.currency.to_string(),
+            dispute_stage: api_models::enums::DisputeStage::Dispute,
+            connector_dispute_id: webhook_object.payme_sale_id,
```

Comment:
Should we confirm with connector if `payme_sale_id` is chargeback_id or connector_transaction_id?